dotCMS/src/integration-test/java/com/dotmarketing/portlets/workflows/business/WorkflowAPITest.java

```diff
@@ -761,6 +761,13 @@ public void findActions() throws DotDataException, DotSecurityException {
         //check valid action for
         actions = workflowAPI.findActions(steps, reviewerUser);
         assertTrue(null != actions && actions.size() == 2);
+
+        actions = workflowAPI.findActions(workflowScheme1Step2, reviewer);
+        assertTrue(null != actions && actions.size() == 1);
```

Comment:
this kind of things is better to separated on two assert
assertNotNull
assertEquals (2, actions.size);

Give you more info about what is the error